contrib/brl/bbas/bdgl/bdgl_curve_algs.cxx

```diff
@@ -570,19 +570,18 @@ bdgl_curve_algs::match_intersection(vdgl_digital_curve_sptr const& dc,
   vgl_homg_point_2d<double> rph(ref_point.x(), ref_point.y());
   double dist = 1e10;
   bool found_valid_intersection = false;
-  for (std::vector<double>::iterator iit = indices.begin();
-       iit != indices.end(); iit++)
+  for (double & indice : indices)
```

Comment:
If it is not too much of a problem, rename `indice` into `index` as it appears a lot.

Also this commit's name (`STYLE: Use reange-baed loops from C++11`) has spelling errors and commit message has a cut-off sentence.

Reply:
done in all 4 files